### ISSUE TITLE
flutter install: Library not loaded libzip-flutter/lib/libzip.5.dylib

### ISSUE THREAD

**Issue by maxlapides** [flutter/flutter#42877]
When I run `flutter install`, here's what I see:

```
❯ flutter install -d REDACTED
Installing com.tonal.mobile to Max's iPhone...
Process exited abnormally:
dyld: Library not loaded: /b/s/w/ir/k/homebrew/opt/libzip-flutter/lib/libzip.5.dylib
  Referenced from: /Users/maxlapides/flutter/bin/cache/artifacts/ideviceinstaller/ideviceinstaller
  Reason: image not found
Install failed
```

I also ran `flutter install` with the `-v` flag if this is useful:

<details><summary>Verbose log</summary>
<p>

```
❯ flutter install -d REDACTED -v
[  +25 ms] executing: [/Users/maxlapides/flutter/] git -c log.showSignature=false log -n 1 --pretty=format:%H
[  +28 ms] Exit code 0 from: git -c log.showSignature=false log -n 1 --pretty=format:%H
[        ] e70236e36ce1d32067dc68eb55519ec3e14b6b01
[        ] executing: [/Users/maxlapides/flutter/] git describe --match v*.*.* --first-parent --long --tags
[   +8 ms] Exit code 0 from: git describe --match v*.*.* --first-parent --long --tags
[        ] v1.10.7-0-ge70236e36
[   +7 ms] executing: [/Users/maxlapides/flutter/] git rev-parse --abbrev-ref --symbolic @{u}
[   +9 ms] Exit code 0 from: git rev-parse --abbrev-ref --symbolic @{u}
[        ] origin/beta
[        ] executing: [/Users/maxlapides/flutter/] git ls-remote --get-url origin
[   +7 ms] Exit code 0 from: git ls-remote --get-url origin
[        ] https://github.com/flutter/flutter.git
[  +49 ms] executing: [/Users/maxlapides/flutter/] git rev-parse --abbrev-ref HEAD
[   +8 ms] Exit code 0 from: git rev-parse --abbrev-ref HEAD
[        ] beta
[   +6 ms] executing: sw_vers -productName
[  +19 ms] Exit code 0 from: sw_vers -productName
[        ] Mac OS X
[        ] executing: sw_vers -productVersion
[  +20 ms] Exit code 0 from: sw_vers -productVersion
[        ] 10.15
[        ] executing: sw_vers -buildVersion
[  +20 ms] Exit code 0 from: sw_vers -buildVersion
[        ] 19A583
[  +31 ms] executing: /usr/bin/xcode-select --print-path
[  +12 ms] Exit code 0 from: /usr/bin/xcode-select --print-path
[        ] /Applications/Xcode.app/Contents/Developer
[   +4 ms] executing: /usr/bin/xcodebuild -version
[ +246 ms] Exit code 0 from: /usr/bin/xcodebuild -version
[   +3 ms] Xcode 11.1
           Build version 11A1027
[  +61 ms] executing: /Users/maxlapides/Library/Android/sdk/platform-tools/adb devices -l
[   +8 ms] Exit code 0 from: /Users/maxlapides/Library/Android/sdk/platform-tools/adb devices -l
[        ] List of devices attached
[  +22 ms] executing: /Users/maxlapides/flutter/bin/cache/artifacts/libimobiledevice/idevice_id -h
[ +307 ms] /usr/bin/xcrun simctl list --json devices
[ +164 ms] Artifact Instance of 'AndroidMavenArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[   +2 ms] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[   +2 ms] Artifact Instance of 'WindowsEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[  +39 ms] executing: /usr/bin/plutil -convert json -o - /Users/maxlapides/Repositories/mobile/ios/Runner/Info.plist
[   +9 ms] Exit code 0 from: /usr/bin/plutil -convert json -o -
/Users/maxlapides/Repositories/mobile/ios/Runner/Info.plist
[   +3 ms]
{"CFBundleName":"Tonal","UILaunchStoryboardName":"LaunchScreen","ITSAppUsesNonExemptEncryption":false,"LSApplicationQu
eriesSchemes":["fbapi","fb-messenger-share-api","fbauth2","fbshareextension"],"CFBundleDevelopmentRegion":"en","CFBund
leVersion":"91","NSCameraUsageDescription":"Upload profile avatar
photos","UIBackgroundModes":["remote-notification"],"NSHealthUpdateUsageDescription":"Synchronize your Tonal activity
and profile with
HealthKit","CFBundlePackageType":"APPL","CFBundleShortVersionString":"0.0.1","UIMainStoryboardFile":"Main","NSHealthSh
areUsageDescription":"Your Tonal profile will automatically be updated with the latest information from
HealthKit","NSAppTransportSecurity":{"NSAllowsArbitraryLoads":true},"CFBundleInfoDictionaryVersion":"6.0","CFBundleExe
cutable":"$(EXECUTABLE_NAME)","FacebookAppID":"2383478315000318","FacebookDisplayName":"Tonal","CFBundleURLTypes":[{"C
FBundleTypeRole":"Editor","CFBundleURLName":"tonalmobileapp","CFBundleURLSchemes":["https"]},{"CFBundleURLSchemes":["f
b2383478315000318"]}],"CFBundleIdentifier":"$(PRODUCT_BUNDLE_IDENTIFIER)","NSPhotoLibraryAddUsageDescription":"Your
photos library is used to save and upload Tonal-related
content","CFBundleSignature":"????","UISupportedInterfaceOrientations~ipad":["UIInterfaceOrientationPortrait","UIInter
faceOrientationPortraitUpsideDown","UIInterfaceOrientationLandscapeLeft","UIInterfaceOrientationLandscapeRight"],"LSRe
quiresIPhoneOS":true,"CFBundleDisplayName":"Tonal","NSLocationAlwaysUsageDescription":"Your location is used to
improve the quality of push
notifications","UISupportedInterfaceOrientations":["UIInterfaceOrientationPortrait"],"NSPhotoLibraryUsageDescription":
"Your photos library is used to save and upload Tonal-related
content","UIViewControllerBasedStatusBarAppearance":true,"NSLocationWhenInUseUsageDescription":"Your location is used
to improve the quality of push notifications"}
[  +15 ms] Installing com.tonal.mobile to Max's iPhone...
[   +3 ms] executing: /Users/maxlapides/flutter/bin/cache/artifacts/ideviceinstaller/ideviceinstaller --list-apps
[  +18 ms] dyld: Library not loaded: /b/s/w/ir/k/homebrew/opt/libzip-flutter/lib/libzip.5.dylib
             Referenced from: /Users/maxlapides/flutter/bin/cache/artifacts/ideviceinstaller/ideviceinstaller
             Reason: image not found
[   +5 ms] executing: /Users/maxlapides/flutter/bin/cache/artifacts/ideviceinstaller/ideviceinstaller -i
build/ios/iphoneos/Runner.app
[  +17 ms] dyld: Library not loaded: /b/s/w/ir/k/homebrew/opt/libzip-flutter/lib/libzip.5.dylib
             Referenced from: /Users/maxlapides/flutter/bin/cache/artifacts/ideviceinstaller/ideviceinstaller
             Reason: image not found
[   +1 ms] Process exited abnormally:
           dyld: Library not loaded: /b/s/w/ir/k/homebrew/opt/libzip-flutter/lib/libzip.5.dylib
             Referenced from: /Users/maxlapides/flutter/bin/cache/artifacts/ideviceinstaller/ideviceinstaller
             Reason: image not found
[  +16 ms] "flutter install" took 1,069ms.
Install failed

#0      throwToolExit (package:flutter_tools/src/base/common.dart:28:3)
#1      InstallCommand.runCommand (package:flutter_tools/src/commands/install.dart:45:7)
<asynchronous suspension>
#2      FlutterCommand.verifyThenRunCommand (package:flutter_tools/src/runner/flutter_command.dart:553:18)
#3      _asyncThenWrapperHelper.<anonymous closure> (dart:async-patch/async_patch.dart:71:64)
#4      _rootRunUnary (dart:async/zone.dart:1132:38)
#5      _CustomZone.runUnary (dart:async/zone.dart:1029:19)
#6      _FutureListener.handleValue (dart:async/future_impl.dart:137:18)
#7      Future._propagateToListeners.handleValueCallback (dart:async/future_impl.dart:678:45)
#8      Future._propagateToListeners (dart:async/future_impl.dart:707:32)
#9      Future._completeWithValue (dart:async/future_impl.dart:522:5)
#10     Future._asyncComplete.<anonymous closure> (dart:async/future_impl.dart:552:7)
#11     _rootRun (dart:async/zone.dart:1124:13)
#12     _CustomZone.run (dart:async/zone.dart:1021:19)
#13     _CustomZone.runGuarded (dart:async/zone.dart:923:7)
#14     _CustomZone.bindCallbackGuarded.<anonymous closure> (dart:async/zone.dart:963:23)
#15     _microtaskLoop (dart:async/schedule_microtask.dart:41:21)
#16     _startMicrotaskLoop (dart:async/schedule_microtask.dart:50:5)
#17     _runPendingImmediateCallback (dart:isolate-patch/isolate_patch.dart:116:13)
#18     _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:173:5)
```

</p>
</details>

I believe this problem has been referenced in some other issues, but I'm not sure if these are the same issue that I'm experiencing:

- https://github.com/flutter/flutter/issues/42543#issuecomment-542635604
- https://github.com/flutter/flutter/issues/29932

This issue has been reproduced on other computers attempting to install on other mobile devices.

```
❯ flutter doctor -v
[✓] Flutter (Channel beta, v1.10.7, on Mac OS X 10.15 19A583, locale en-US)
    • Flutter version 1.10.7 at /Users/maxlapides/flutter
    • Framework revision e70236e36c (2 weeks ago), 2019-10-02 09:32:30 -0700
    • Engine revision 9e6314d348
    • Dart version 2.6.0 (build 2.6.0-dev.0.0 1103600280)


[✓] Android toolchain - develop for Android devices (Android SDK version 28.0.3)
    • Android SDK at /Users/maxlapides/Library/Android/sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-28, build-tools 28.0.3
    • ANDROID_HOME = /Users/maxlapides/Library/Android/sdk
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_202-release-1483-b49-5587405)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 11.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 11.1, Build version 11A1027
    • CocoaPods version 1.8.3

[✓] Android Studio (version 3.5)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 40.2.2
    • Dart plugin version 191.8423
    • Java version OpenJDK Runtime Environment (build 1.8.0_202-release-1483-b49-5587405)

[✓] VS Code (version 1.39.2)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.5.1

[✓] Connected device (2 available)
    • Max's iPhone  • REDACTED            • ios • iOS 13.1.2
    • iPhone 11 Pro • 8D361F98-6571-4CB0-A6B9-BB3C8A36FBB9 • ios • com.apple.CoreSimulator.SimRuntime.iOS-13-1
      (simulator)

• No issues found!
```

**Comment by BondarenkoStas** [comment]
@maxlapides by other devices do you mean only ios, or it also crashes with android?

**Comment by maxlapides** [comment]
@BondarenkoStas Only iOS. This issue does not occur when installing on Android devices.

**Comment by Kiruel** [comment]
Happen on when I try to `flutter install` on my iPhone XR.

```
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel master, v1.10.13-pre.10, on Mac OS X 10.15 19A583, locale en-FR)
[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.2)
[✓] Xcode - develop for iOS and macOS (Xcode 11.0)
[✓] Chrome - develop for the web
[✓] Android Studio (version 3.5)
[✓] VS Code (version 1.39.2)
[✓] Connected device (4 available)

• No issues found!
```

The error logs:
```
flutter install
Installing com.example.weatherTdd to iPhone de USER...
Process exited abnormally:
dyld: Library not loaded: /b/s/w/ir/k/homebrew/opt/libzip-flutter/lib/libzip.5.dylib
  Referenced from: /Users/USER/Documents/software/flutter/bin/cache/artifacts/ideviceinstaller/ideviceinstaller
  Reason: image not found
Install failed
```

**Comment by Kiruel** [comment]
On channel stable another error happen:
```
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, v1.9.1+hotfix.5, on Mac OS X 10.15 19A583, locale en-FR)
[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.2)
[✓] Xcode - develop for iOS and macOS (Xcode 11.0)
[✓] Android Studio (version 3.5)
[✓] VS Code (version 1.39.2)
[✓] Connected device (2 available)

• No issues found!
```

The error when try to `flutter install`:

```
[  +17 ms] executing: [/Users/USER/Documents/software/flutter/] git log -n 1 --pretty=format:%H
[  +34 ms] Exit code 0 from: git log -n 1 --pretty=format:%H
[        ] 1aedbb1835bd6eb44550293d57d4d124f19901f0
[        ] executing: [/Users/USER/Documents/software/flutter/] git describe --match v*.*.* --first-parent --long --tags
[  +11 ms] Exit code 0 from: git describe --match v*.*.* --first-parent --long --tags
[        ] v1.9.1+hotfix.5-0-g1aedbb183
[   +6 ms] executing: [/Users/USER/Documents/software/flutter/] git rev-parse --abbrev-ref --symbolic @{u}
[  +12 ms] Exit code 0 from: git rev-parse --abbrev-ref --symbolic @{u}
[        ] origin/stable
[        ] executing: [/Users/USER/Documents/software/flutter/] git ls-remote --get-url origin
[  +10 ms] Exit code 0 from: git ls-remote --get-url origin
[        ] https://github.com/flutter/flutter.git
[  +49 ms] executing: [/Users/USER/Documents/software/flutter/] git rev-parse --abbrev-ref HEAD
[  +10 ms] Exit code 0 from: git rev-parse --abbrev-ref HEAD
[        ] stable
[  +15 ms] executing: /Users/USER/Documents/software/flutter/bin/cache/artifacts/libimobiledevice/idevice_id -h
[ +196 ms] /usr/bin/xcrun simctl list --json devices
[ +118 ms] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[   +2 ms] Artifact Instance of 'WindowsEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[  +28 ms] executing: /usr/bin/plutil -convert json -o - /Users/USER/Documents/flutter_projects/weather_tdd/ios/Runner/Info.plist
[   +8 ms] Exit code 0 from: /usr/bin/plutil -convert json -o - /Users/USER/Documents/flutter_projects/weather_tdd/ios/Runner/Info.plist
[        ]
{"CFBundleShortVersionString":"$(FLUTTER_BUILD_NAME)","CFBundleIdentifier":"$(PRODUCT_BUNDLE_IDENTIFIER)","CFBundleInfoDictionaryVersion":"6.0","UIMainStoryboardFile":"Main","CFBundleVersion":"$(FLUTTER_BUILD_NUM
BER)","UILaunchStoryboardName":"LaunchScreen","CFBundleExecutable":"$(EXECUTABLE_NAME)","LSRequiresIPhoneOS":true,"UISupportedInterfaceOrientations":["UIInterfaceOrientationPortrait","UIInterfaceOrientationLandsc
apeLeft","UIInterfaceOrientationLandscapeRight"],"UIViewControllerBasedStatusBarAppearance":false,"CFBundleSignature":"????","CFBundlePackageType":"APPL","CFBundleDevelopmentRegion":"$(DEVELOPMENT_LANGUAGE)","UIS
upportedInterfaceOrientations~ipad":["UIInterfaceOrientationPortrait","UIInterfaceOrientationPortraitUpsideDown","UIInterfaceOrientationLandscapeLeft","UIInterfaceOrientationLandscapeRight"],"CFBundleName":"weath
er_tdd"}
[  +14 ms] Installing com.example.APPLICATION to iPhone de USER...
[   +2 ms] executing: /Users/USER/Documents/software/flutter/bin/cache/artifacts/ideviceinstaller/ideviceinstaller --list-apps
[  +30 ms] dyld: Library not loaded: /b/s/w/ir/k/homebrew/opt/libzip-flutter/lib/libzip.5.dylib
             Referenced from: /Users/USER/Documents/software/flutter/bin/cache/artifacts/ideviceinstaller/ideviceinstaller
             Reason: image not found
[   +3 ms] executing: /Users/USER/Documents/software/flutter/bin/cache/artifacts/ideviceinstaller/ideviceinstaller -i build/ios/iphoneos/Runner.app
[   +6 ms] dyld: Library not loaded: /b/s/w/ir/k/homebrew/opt/libzip-flutter/lib/libzip.5.dylib
             Referenced from: /Users/USER/Documents/software/flutter/bin/cache/artifacts/ideviceinstaller/ideviceinstaller
             Reason: image not found
[   +9 ms] "flutter install" took 668ms.
Install failed

#0      throwToolExit (package:flutter_tools/src/base/common.dart:28:3)
#1      InstallCommand.runCommand (package:flutter_tools/src/commands/install.dart:44:7)
<asynchronous suspension>
#2      FlutterCommand.verifyThenRunCommand (package:flutter_tools/src/runner/flutter_command.dart:490:18)
#3      _asyncThenWrapperHelper.<anonymous closure> (dart:async-patch/async_patch.dart:71:64)
#4      _rootRunUnary (dart:async/zone.dart:1132:38)
#5      _CustomZone.runUnary (dart:async/zone.dart:1029:19)
#6      _FutureListener.handleValue (dart:async/future_impl.dart:137:18)
#7      Future._propagateToListeners.handleValueCallback (dart:async/future_impl.dart:678:45)
#8      Future._propagateToListeners (dart:async/future_impl.dart:707:32)
#9      Future._completeWithValue (dart:async/future_impl.dart:522:5)
#10     Future._asyncComplete.<anonymous closure> (dart:async/future_impl.dart:552:7)
#11     _rootRun (dart:async/zone.dart:1124:13)
#12     _CustomZone.run (dart:async/zone.dart:1021:19)
#13     _CustomZone.runGuarded (dart:async/zone.dart:923:7)
#14     _CustomZone.bindCallbackGuarded.<anonymous closure> (dart:async/zone.dart:963:23)
#15     _microtaskLoop (dart:async/schedule_microtask.dart:41:21)
#16     _startMicrotaskLoop (dart:async/schedule_microtask.dart:50:5)
#17     _runPendingImmediateCallback (dart:isolate-patch/isolate_patch.dart:116:13)
#18     _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:173:5)
```

**Comment by maxlapides** [comment]
Tim on the Flutter team reached out to me with some questions regarding this issue. Here are my answers:

1. Does it work if you do: flutter run? please provide the output of “-v” if it also fails

**Yes, `flutter run` works on my physical iOS device.**

2. Does it work if you install on a simulator and not on your iPhone?

**Yes, `flutter run` works on the iOS simulator. It is not possible to `flutter install` on an iOS simulator.**

3. Would you happen to have another iPhone (or even one on lower than iOS 13) that you can test on?

**I have tested this by attempting to `flutter install` on a different iPhone with a different computer and experienced the same issue. Unfortunately we no longer have any iOS 12 test devices.**

4. Also, per /flutter/issues/42543#issuecomment-542820599, can you also answer that question?
   - Flutter isn't distributed with homebrew, did you install some unofficial installation?

**No, my coworkers and I have all installed Flutter using the official instructions (https://flutter.dev/docs/get-started/install/macos#get-the-flutter-sdk)**

5. Can you also check the status of your /homebrew?
   - brew update & brew doctor (/Homebrew/brew/issues/575)

**Here are the full results:**

<details><summary>brew update && brew doctor</summary>
<p>

```
❯ brew update && brew doctor
Already up-to-date.
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: "config" scripts exist outside your system or Homebrew directories.
`./configure` scripts often look for *-config scripts to determine if
software packages are installed, and which additional flags to use when
compiling and linking.

Having additional scripts in your path can confuse software installed via
Homebrew if the config script overrides a system or Homebrew-provided
script of the same name. We found the following "config" scripts:
  /Applications/Postgres.app/Contents/Versions/latest/bin/gdal-config

Warning: Unbrewed header files were found in /usr/local/include.
If you didn't put them there on purpose they could cause problems when
building Homebrew formulae, and may need to be deleted.

Unexpected header files:
  /usr/local/include/node/android-ifaddrs.h
  /usr/local/include/node/pthread-barrier.h
  /usr/local/include/node/stdint-msvc2008.h
  /usr/local/include/node/tree.h
  /usr/local/include/node/uv-aix.h
  /usr/local/include/node/uv-bsd.h
  /usr/local/include/node/uv-darwin.h
  /usr/local/include/node/uv-errno.h
  /usr/local/include/node/uv-linux.h
  /usr/local/include/node/uv-os390.h
  /usr/local/include/node/uv-posix.h
  /usr/local/include/node/uv-sunos.h
  /usr/local/include/node/uv-threadpool.h
  /usr/local/include/node/uv-unix.h
  /usr/local/include/node/uv-version.h
  /usr/local/include/node/uv-win.h
  /usr/local/include/node/v8-inspector-protocol.h
  /usr/local/include/node/v8-inspector.h

Warning: Some installed formulae are missing dependencies.
You should `brew install` the missing dependencies:
  brew install node

Run `brew missing` for more details.
```

</p>
</details>

**Comment by jmagman** [comment]
I see this on master.
\cc @christopherfujino

**Comment by christopherfujino** [comment]
investigating now

**Comment by jmagman** [comment]
https://github.com/flutter/flutter/issues/42543#issuecomment-542820599
> Flutter isn't distributed with homebrew, did you install some unofficial installation?
> 
> ```
> [   +4 ms] dyld: Library not loaded: /b/s/w/ir/k/homebrew/opt/libzip-flutter/lib/libzip.5.dylib
> ```

Well I was wrong, this is totally reproducible.  Mea culpa, @tottakai.

**Comment by christopherfujino** [comment]
i'm working on a fix for this

**Comment by christopherfujino** [comment]
Update, my fix for this is code complete, I'm just blocked on being able to codesign the libzip binary, then I will be able to land to master.

**Comment by mleonhard** [comment]
Thank you!  Now I can build for iOS 10.3.

**Comment by murphycrosby** [comment]
Wha't the fix for this?  This is my flutter doctor:

[✓] Flutter (Channel stable, v1.9.1+hotfix.6, on Mac OS X 10.14.6 18G95, locale en-US)
    • Flutter version 1.9.1+hotfix.6 at /Users/mhcrosb1/libs/flutter
    • Framework revision 68587a0916 (9 weeks ago), 2019-09-13 19:46:58 -0700
    • Engine revision b863200c37
    • Dart version 2.5.0


[✓] Android toolchain - develop for Android devices (Android SDK version 28.0.3)
    • Android SDK at /Users/mhcrosb1/Library/Android/sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-28, build-tools 28.0.3
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1248-b01)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 11.2.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 11.2.1, Build version 11B500
    • CocoaPods version 1.7.5

[✓] Android Studio (version 3.3)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 33.3.1
    • Dart plugin version 182.5215
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1248-b01)

[✓] IntelliJ IDEA Community Edition (version 2019.2.4)
    • IntelliJ at /Applications/IntelliJ IDEA CE.app
    • Flutter plugin version 41.1.4
    • Dart plugin version 192.7402

[✓] VS Code (version 1.40.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.6.0

[✓] Connected device (2 available)
    • Murphy’s XsMax    • 00008020-000224512291002E            • ios • iOS 13.2.2
    • iPhone 11 Pro Max • 21E9D80B-F3DD-4C1B-A23C-461C082147C0 • ios • com.apple.CoreSimulator.SimRuntime.iOS-13-2
      (simulator)

• No issues found!

**Comment by christopherfujino** [comment]
@murphycrosby This was fixed in https://github.com/flutter/flutter/pull/43677, which landed in version 1.10.15. You can get the fix today by checking out the dev branch. The next beta and stable releases should pick it up as well.
